assets/multiqc_config.yml

```diff
@@ -1,7 +1,8 @@
 report_comment: >
+
   This report has been generated by the <a href="https://github.com/nf-core/airrflow/releases/tag/4.0" target="_blank">nf-core/airrflow</a>
   analysis pipeline. For information about how to interpret these results, please see the
-  <a href="https://nf-co.re/airrflow" target="_blank">documentation</a>.
+  <a href="https://nf-co.re/airrflow/4.1.0/docs/output" target="_blank">documentation</a>.
```

Comment:
Should 4.1.0 be 4.2.0? Similarly, in line 3, should 4.0 be 4.2.0?

Reply:
I'll bump all versions to 4.2.0 in the next PR, I haven't bumped them yet